app/models/cms/base_page.rb

```diff
@@ -23,4 +23,40 @@ def parent_sections
     sections.reverse
   end
 
+  def hidden?
+    published.nil?
+  end
+
+  def to_xml(options = {})
+    xml = options[:builder] || Nokogiri::XML::Builder.new
+
+    xml.__send__(options.fetch(:root, :page)) do |x|
```

Comment:
I copied it from https://github.com/3scale/porta/blob/91c45c2e8328eef16f85ebb60c116bac598fb977/app/models/cms/partial.rb#L22

I haven't checked it, but I assume it's because if we call `send` it will create a `<send>` tag in the response. I tried calling `public_send` and `__public_send__` and it returns a weird error, too many levels of meta, apparently 🤕